bin/dlang/install

```diff
@@ -14,7 +14,12 @@ case "$DMD" in
     dmd*   ) PKG= ;;
     1.*    ) PKG="dmd1=$DMD-$DIST" ;;
     2.*.s* ) PKG="dmd-transitional=$DMD-$DIST" ;;
-    2.*    ) PKG="dmd-bin=$DMD libphobos2-dev=$DMD" ;;
+    2.*    ) if [ $(echo $DMD | cut -d. -f2) -ge 077 ]; then
+                PKG="dmd-compiler=$DMD dmd-tools=$DMD libphobos2-dev=$DMD"
+             else
+                PKG="dmd-bin=$DMD libphobos2-dev=$DMD"
+             fi
+             echo $PKG ;;
```

Comment:
I guess this debug `echo` should be gone now :)

Reply:
Updated :P